pkg/ocicni/ocicni.go

```diff
@@ -857,3 +857,27 @@ func (plugin *cniNetworkPlugin) Status() error {
 	}
 	return nil
 }
+
+// Name returns the network identifier
+func (n *CNINetwork) Name() string {
+	return n.name
+}
+
+// FilePath returns the network file path on disk
+func (n *CNINetwork) FilePath() string {
+	return n.filePath
+}
+
+// Config returns the CNI network config
+func (n *CNINetwork) Config() *libcni.NetworkConfigList {
```

Comment:
Any particular reason callers would want the network config list? What kind of use-cases would there be for it?